src/debug/Stream.h

```diff
@@ -207,6 +207,11 @@ void ResyncDebugLog(FILE *newDestination);
         } \
    } while (/*CONSTCOND*/ 0)
 
+/// Convenience wrapper for displaying protocol message flow in cache.log
+#define traceProtocol(SECTION, LABEL, ID, MESSAGE) \
+    debugs(SECTION, 2, LABEL << ": " << ID << \
```

Comment:
Please protect macro arguments (where possible):

```suggestion
    debugs((SECTION), 2, (LABEL) << ": " << ID << \
```